src/products/executable_generators.jl

```diff
@@ -2,6 +2,20 @@ function declare_old_executable_product(product_name)
     path_name = Symbol(string(product_name, "_path"))
     return quote
         # This is the old-style `withenv()`-based function
+        """
+            $($product_name)(f::Function; adjust_PATH::Bool=true, adjust_LIBPATH::Bool=true)
+
+        An `ExecutableProduct` wrapper that supports the execution of $($product_name).
+
+        # Example
+        ```julia
+        $($product_name)() do exe
+            run(`\$exe \$arguments`)
+        end
+        ```
+
+        !!! compat "Julia 1.0"
```

Comment:
```suggestion
        !!! compat "Julia 1.3"
```

Reply:
Ah, also good to know. I was just going off of the `Project.toml`